pyflakes/checker.py

```diff
@@ -2036,24 +1981,14 @@ def LAMBDA(self, node):
         args = []
         annotations = []
 
-        if PY2:
-            def addArgs(arglist):
-                for arg in arglist:
-                    if isinstance(arg, ast.Tuple):
-                        addArgs(arg.elts)
-                    else:
-                        args.append(arg.id)
-            addArgs(node.args.args)
-            defaults = node.args.defaults
-        else:
-            if PY38_PLUS:
-                for arg in node.args.posonlyargs:
-                    args.append(arg.arg)
-                    annotations.append(arg.annotation)
-            for arg in node.args.args + node.args.kwonlyargs:
+        if PY38_PLUS:
+            for arg in node.args.posonlyargs:
                 args.append(arg.arg)
                 annotations.append(arg.annotation)
-            defaults = node.args.defaults + node.args.kw_defaults
+        for arg in node.args.args + node.args.kwonlyargs:
+            args.append(arg.arg)
+            annotations.append(arg.annotation)
+        defaults = node.args.defaults + node.args.kw_defaults
 
         # Only for Python3 FunctionDefs
         is_py3_func = hasattr(node, 'returns')
```

Comment:
this should always be true

Reply:
When I tried https://github.com/hugovk/pyflakes/commit/79ff7c04aa2d8432d2ce8c8d9e7c48ff3124c8af, it failed:

```
  File "/home/runner/work/pyflakes/pyflakes/pyflakes/checker.py", line 1994, in LAMBDA
    annotations.append(node.returns)
AttributeError: 'Lambda' object has no attribute 'returns'
```

https://github.com/hugovk/pyflakes/actions/runs/403118601

Reply:
hmmm perhaps `LAMBDA` should go through a different code path 🤔